Datatable/AbstractDatatable.php

```diff
@@ -151,10 +152,10 @@ abstract class AbstractDatatable implements DatatableInterface
     public function __construct(
         AuthorizationCheckerInterface $authorizationChecker,
         TokenStorageInterface $securityToken,
-        TranslatorInterface $translator,
+        object $translator,
```

Comment:
The Composer manifest says PHP 7.1 is the minimum version, so the `object` typehint isn't available.

Reply:
Cheers for the feedback - now changed :)